ext/opcache/tests/preload_unresolved_prop_type.phpt

```diff
@@ -11,7 +11,12 @@ require_once('skipif.inc');
 if (PHP_OS_FAMILY == 'Windows') die('skip Preloading is not supported on Windows');
 ?>
 --FILE--
+<?php
+class Unknown {
+}
+$x = new Test;
+$x->prop = new Unknown;
+?>
 ===DONE===
---EXPECTF--
-Warning: Can't preload class Test with unresolved property types in %s on line %d
+--EXPECT--
```

Comment:
This change in expectation seems weird: is the `--FILE--` section mostly ignored and only here for representative purposes? Does the Warning still get emitted in some cases, after this patch?

Reply:
Now opcache may preload classes with unresolved properties without problems.